pkg/generate/command.go

```diff
@@ -0,0 +1,35 @@
+package generate
+
+import (
+	"fmt"
+	"log"
+	"github.com/budimanjojo/talhelper/pkg/config"
+	"github.com/budimanjojo/talhelper/pkg/talos"
+)
+
+func GenerateCommand(c *config.TalhelperConfig, outDir string, generateForNode string, generateApply bool, generateUpgrade bool) error {
```

Comment:
This function shouldn't return `nil` when there's error and instead of using `log.Fatalf` return the error instead.